pac4j-kerberos/src/main/java/org/pac4j/kerberos/profile/KerberosProfile.java

```diff
@@ -0,0 +1,16 @@
+package org.pac4j.kerberos.profile;
+
+import org.ietf.jgss.GSSContext;
+import org.pac4j.core.profile.CommonProfile;
+
+/**
+ * Represents a user profile based on a Kerberos authentication.
+ *
+ * @author Garry Boyce
+ * @since 2.1.0
+ */
+public class KerberosProfile extends CommonProfile {
+
+    private static final long serialVersionUID = -1388563485891552197L;
+    public GSSContext gssContext = null;
```

Comment:
added optional `gssContext` for more advanced usage, if any.
if `SunJaasKerberosTicketValidator.setHoldOnToGSSContext(true)` was called, when this would allow subsequent encrypt/decrypt operations.  see https://github.com/pac4j/pac4j/pull/919/files#diff-c6ef70675c7cb3b2df3dde625bac09ecR124